apps/native-component-list/src/screens/Location/LocationScreens.ts

```diff
@@ -1,9 +1,9 @@
-import LocationScreen from './LocationScreen';
-import GeofencingScreen from './GeofencingScreen';
 import BackgroundLocationMapScreen from './BackgroundLocationMapScreen';
+import GeofencingScreen from './GeofencingScreen';
+import LocationScreen from './LocationScreen';
 
 export default {
-  Location: { screen: LocationScreen },
-  Geofencing: { screen: GeofencingScreen },
-  BackgroundLocationMap: { screen: BackgroundLocationMapScreen },
+  Location: LocationScreen,
```

Comment:
This is moved from https://github.com/expo/expo/pull/8857 v5 doesn't support screen and v4 has it as optional